src/test/java/boot/bootdemo/service/impl/ParserServiceImplTest.java

```diff
@@ -0,0 +1,58 @@
+package boot.bootdemo.service.impl;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import boot.bootdemo.dto.ReviewDto;
+import boot.bootdemo.service.CsvReaderService;
+import boot.bootdemo.service.ParserService;
+import java.time.Instant;
+import java.time.LocalDateTime;
+import java.time.ZoneId;
+import org.junit.Assert;
+import org.junit.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+
+@SpringBootTest
+public class ParserServiceImplTest {
+    private static final String FILE_NAME = "src/main/resources/line.csv";
+    private static final String WRONG_FORMAT = "src/main/resources/wrong_format.csv";
```

Comment:
```suggestion
    private static final String FILE_NAME = "src/test/resources/line.csv";
    private static final String WRONG_FORMAT = "src/test/resources/wrong_format.csv";
```

resources for testing should better lay in `src/test` folder